microdata_validator/metadata_model.py

```diff
@@ -90,14 +90,14 @@ class ValueDomain:
 
     def __init__(self, value_domain_dict: dict):
         self.is_enumerated_value_domain = (
-            "codeList" in value_domain_dict
-            and "description" not in value_domain_dict
-            and "unitOfMeasure" not in value_domain_dict
+                "codeList" in value_domain_dict
```

Comment:
why the double indent 🤔 ? Is this an jetbrains thing?

Reply:
Yes, Intellij has continuation indent set to 8 by default. I have changed it to 4 to be aligned with the convention here 👍